README.md

```diff
@@ -2,6 +2,10 @@
 
 A current overview of Notary v2, including a collection of requirements & scenarios, framing the scope of the Notary v2 project.
 
+## Project Status
+
+The Notary v2 project is in early development. Please refer to the [milestones](https://github.com/notaryproject/notaryproject/milestones) or [attend](#contributing-&-conversations) the weekly meetings for details on the roadmap.
```

Comment:
```suggestion
The Notary v2 project is in early development and design documents should not be considered final. Please refer to the [milestones](https://github.com/notaryproject/notaryproject/milestones) or [attend](#contributing-&-conversations) the weekly meetings for details on the roadmap.
```

Reply:
I also left comments on some of the project milestones. The first one was targeted to be done this month, and as we are still working on requirements I think it should be pushed to the end of the year at the earliest.